src/frontend/citizen-portal/src/app/shared/components/resolution-header/resolution-header.component.html

```diff
@@ -1,13 +1,14 @@
 <h2 class="my-2">
-  <div fxLayout="row">
-    <div fxFlex="25">
+  <div fxLayout="row wrap">
+    <div fxFlex.gt-md="25" fxFlex.lt-lg="100">
       <small class="me-2 text-nowrap">Ticket #:</small>
       <b class="text-nowrap">{{ ticketNumber }}</b>
     </div>
-    <div fxFlex style="text-align: end;">
+    <div fxFlex>
+      <div fxFlex.gt-md></div>
       <small class="me-2 text-nowrap">Violation time:</small>
       <b class="text-nowrap">{{ time }}</b>
-      <b *ngIf="date" fxHide.gt-md="true" fxShow.lt-lg="true"> {{ date }}</b>
+      <b *ngIf="date" fxHide.gt-md="true" fxShow.lt-lg="true">&nbsp;{{ date }}</b>
```

Comment:
Are spaces better than using css to add a margin/padding?

Reply:
Because it is only required 1 "space" in between words, not using for layout